boards/riscv/beagle_v_starlight/Kconfig.board

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_BEAGLE_V_STARLIGHT
```

Comment:
I'm not sure how much of this naming dirty laundry to make visible to end users, but we'll have at least two "BeagleV Starlight" boards: one with 7100 and one with 7110. Does this naming scheme allow that?

(I'm struggling with this in my own code, too....)

Reply:
beagle_v_starlight_beta sounds lengthy, to be honest. I am not sure about this.

I am kind of under the impression that this board is for development purpose only and there won't be any more boards based on JH7100. So I don't think it makes sense to continue supporting this beta SoC. Instead, the work done on it can be used as a base and when the production SoC comes in, it's ahead of the initial phase of development.

So in short what I am suggesting here is, we can keep the name beagle_v_starlight. when the production board comes in, this port can be migrated to support the production version and we can say goodbye to the beta one.
Just some raw thoughts. @pdp7, can you comment on this?

Reply:
I think the config option should be BEAGLEV_STARLIGHT_JH7100.  I would not use the terms beta or production.  Instead, it is best to use: BeagleV Starlight JH7100 and BeagleV Starlight JH7110.  

When adding support for JH7110 in the future, it may be possible to have an a wildcard config BEAGLEV_STARLIGHT_JH71XX to enable common support and then BEAGLEV_STARLIGHT_JH7100 and BEAGLEV_STARLIGHT_JH7110 depend on  BEAGLEV_STARLIGHT_JH71XX being enabled

Reply:
@rajnesh-kanwal - @pdp7 is the authoritative source ^^ ;-)

uppercase for Kconfig options, and then directly to lowercase for the board name, paths, filenames, etc.